batch.R

```diff
@@ -170,8 +170,8 @@ for(i in 1:nrow(fileDF)) {
   #TODO: model metrics for non-rloadest models
   #combine into DF with row for each model
   #need to extract fitted model so rloadest functions can be used
-  metrics <- summarizeModel(getFittedModel(rloadest5param))
-  write.csv(x = metrics, file = file.path(outputFolder, constitName, "modelMetrics", paste(constitSite, "modelMetrics.csv", sep = "_")), row.names = FALSE)
+  metrics <- summarizeModel(getFittedModel(rloadest5param), constituent = constitName, site = constitSite)
```

Comment:
yeah, i could have communicated that a lot better. sorry for not doing so until now.